packages/contents/src/drivefs.ts

```diff
@@ -11,6 +12,12 @@ export const DRIVE_SEPARATOR = ':';
 const encoder = new TextEncoder();
 const decoder = new TextDecoder('utf-8');
 
+export interface IDriveRequest {
```

Comment:
Maybe we can add a quick docstring to this interface since it's exported?

Or remove the `export` if it doesn't need to be exported?

Reply:
Thank you for the review! I added a docstring and used this interface where we need it